codes/ruby/utils/print_util.rb

```diff
@@ -0,0 +1,21 @@
+=begin
+File: print_util.rb
+Created Time: 2024-03-18
+Author: Xuan Khoa Tu Nguyen (ngxktuzkai2000@gmail.com)
+=end
+
+module Utils
+  class Printer
```

Comment:
Remove the class if it is not necessary.

(In Java, every method should belong to a class)

Reply:
I see, I wrote them based on the Java files, I will update them based on the Python files instead. 

But I see that the method `linked_list_to_list` in `python/modules/list_node.py` is different from the code in other programming languages, which are `getListNode` instead. Which one should I implement for the Ruby?

Reply:
`getListNode()` and `linked_list_to_list()` are both not called in the source code
I'll remove the `getListNode()` for all the languages.